test/postgresql.test.js

```diff
@@ -201,6 +210,50 @@ describe('postgresql connector', function() {
     });
   });
 
+  it('should support creating and updating arrays with default dataType', function(done) {
+    let postId;
+    Post.create({title: 'Updating Arrays', content: 'Content', tags: ['AA', 'AB']})
+      .then((post)=> {
+        postId = post.id;
+        post.should.have.property('tags');
```

Comment:
Nitpick: You can assert the value for `tags` directly as
```js
 post.should.have.property('tags');
 post.tags.should.be.Array();
 post.tags.length.should.eql(2);
 post.tags.should.eql(['AA', 'AB']);
```
so first check property exists, then its type, then its length(optional), then the value. 
Usually these guarantee if test fails, the error message will point to the exact wrong place :)